app/index.html

```diff
@@ -144,6 +146,8 @@
     <script src="bower_components/angular-highlightjs/build/angular-highlightjs.js"></script>
     <script src="bower_components/ng-file-upload/angular-file-upload.js"></script>
     <script src="bower_components/ng-file-upload-shim/angular-file-upload-shim.js"></script>
+    <script src="bower_components/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js"></script>
```

Comment:
Initially time based filtering wasn't needed for transactions/audits but I agree with @devcritter that keeping it consistent will be better. We might just need to check but I cant see any reason why swapping the old one with the new one shouldn't return the same result. The date that will be sent to the API will just contain extra "time" information